dom.bs

```diff
@@ -1026,24 +1049,40 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 </dl>
 
 <p>The
-<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>capture</var>)</code></dfn>
+<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>options</var>)</code></dfn>
 method, when invoked, must run these steps:
 
 <ol>
  <li><p>If <var>callback</var> is null, terminate these steps.
 
+ <li>If <var>options</var> is of type boolean, let <var>capture</var> be
+ <var>options</var> and let <var>passive</var> be false.  Otherwise let
+ <var>capture</var> and <var>passive</var> be the corresponding values in the
+ <var>options</var> {{EventListenerOptions}} dictionary.
```

Comment:
Well, they have to be defined in the algorithm anyway. Since the dictionary is not guaranteed. So the IDL has no meaningful effect. I'd be okay with comments in the IDL saying what it effectively comes down to.